packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.ts

```diff
@@ -65,8 +70,17 @@ export class RecaptchaEnterpriseVerifier {
    *
    * @returns A Promise for a token that can be used to assert the validity of a request.
    */
-  async verify(): Promise<string> {
+  async verify(action: string = 'verify', forceRefresh: boolean = false): Promise<string> {
     async function retrieveSiteKey(auth: AuthInternal): Promise<string> {
+      if (!forceRefresh) {
+        if (auth.tenantId == null && RecaptchaEnterpriseVerifier.agentSiteKey != null) {
+          return Promise.resolve(RecaptchaEnterpriseVerifier.agentSiteKey);
```

Comment:
Since it's an async function, you don't need to return `Promise.resolve()`; the compiler will take care of it. Just return the value directly